apps/studio/src/lib/db/clients/sqlserver.js

```diff
@@ -833,6 +834,15 @@ export async function truncateAllTables(conn) {
   });
 }
 
+export async function dropElement (conn, elementName, typeOfElement, schema) {
+  await runWithConnection(conn, async (connection) => {
+    const connClient = { connection };
+    const sql = `DROP ${typeOfElement} ${wrapIdentifier(schema)}.${wrapIdentifier(elementName)}`
```

Comment:
What if schema isn't provided?

Reply:
Same as above with postgres, will take the same approach.